message_ix_models/tools/costs/config.py

```diff
@@ -55,6 +55,13 @@ class Config:
     #: Model variant for which to project costs.
     module: Literal["energy", "materials"] = "energy"
 
+    #: Use vintages
+    #: If True, for each vintage, the fixed O&M costs will be calculated as a
+    #: ratio of the investment costs, that decreases by the rate of :attr:`fom_rate`.
+    #: If False, the fix_cost is the ratio of the investment cost for each year_act.
+    #: In this case, the fix_cost is the same for all vintages of the same year_act.
+    use_vintages: bool = False
```

Comment:
Nice, clean addition! This is one way using a Config class helps clarify changes to code.

Please check how the docstring renders. I think Sphinx will rewrap the first two lines so they look like "Use vintages If True" (with no period).

Reply:
Thanks! 

I have edited to account for the docstring rendering. 🙂